server/src/modules/contests/contests.service.ts

```diff
@@ -404,7 +406,14 @@ export class ContestsService {
     if (isAdmin && newState === ContestState.Published) {
       this.logger.log(`Publishing contest ${contest.competitionId}...`);
 
-      if (contest.type !== ContestType.WcaComp && contest.participants < C.minCompetitorsForUnofficialCompsAndMeetups) {
+      if (contest.type === ContestType.WcaComp) {
+        const response = await fetch(`https://www.worldcubeassociation.org/api/v0/competitions/${competitionId}/results`);
```

Comment:
This is probably the only way to check if results have been posted, `/competitions/:id` endpoint doesn't have any information about this